### ISSUE TITLE
Is there a way to format particular operand, not all operands?

### ISSUE THREAD

**Issue by 10110111** [aquynh/capstone#457]
For a debugger I'm trying to switch to Capstone I need a way to format a particular operand on demand, to show e.g. messages like

    dword ptr [esi+ecx*4+0x35f34] = 0x21af47 <myFunction>

Reading `capstone.h` and `x86.h`, I've only found the way to print the whole operands string, e.g.

    eax, dword ptr [esi+ecx*4+0x35f34]

It'd be quite hard to parse such strings (especially in AT&T syntax, where comma can separate not only operands, but also SIB components and whatnot), so I'd like to use some better way to get formatted operand.

How do I do it with Capstone?

**Comment by aquynh** [comment]
the detail mode allows you to access to all operands separately, but then you have to construct the operand string yourself.

but otherwise, i dont see any easy way to do what you want.

**Comment by 10110111** [comment]
Could Capstone API be extended to provide this functionality? E.g. adding a member to each operand like `string_form[50]`.
If e.g. I implement this, can it be pulled into the official sources?

**Comment by aquynh** [comment]
this is not as easy as it seems, because sometimes the operands are already mixed in the output at very early phase.

the additional fields do not look nice anyway, and i am wondering if there are many users want this. if this is just useful for very few people, then sorry no i don't want to merge it.

**Comment by 10110111** [comment]
OK, thank you.
